Tests/ScreenPixelAlignmentTests.swift

```diff
@@ -62,17 +62,11 @@ final class ScreenPixelAlignmentTests: XCTestCase {
   // MARK: Approximate equality tests
 
   func testApproximateEquality() {
-    XCTAssert(CGFloat(1.48).isEqual(to: 1.52, threshold: 0.05))
-    XCTAssert(!CGFloat(1.48).isEqual(to: 1.53, threshold: 0.05))
-
-    XCTAssert(CGFloat(1).isEqual(to: 10, threshold: 9))
-    XCTAssert(!CGFloat(1).isEqual(to: 11, threshold: 9))
-
-    XCTAssert(CGFloat(1).isEqual(to: 10, threshold: 9))
-    XCTAssert(!CGFloat(1).isEqual(to: 11, threshold: 9))
-
-    XCTAssert(CGFloat(1.333).isEqual(to: 1.666, threshold: 1 / 3))
-    XCTAssert(!CGFloat(1.332).isEqual(to: 1.666, threshold: 1 / 3))
+    XCTAssert(CGFloat(1.48).isEqual(to: 1.52, screenScale: 2))
+    XCTAssert(!CGFloat(1).isEqual(to: 10, screenScale: 9))
+    XCTAssert(!CGFloat(1).isEqual(to: 10, screenScale: 9))
+    XCTAssert(!CGFloat(1).isEqual(to: 9, screenScale: 9))
+    XCTAssert(!CGFloat(1.333).isEqual(to: 1.666, screenScale: 3))
```

Comment:
what was I thinking - of course these aren't equal. They're literally 1px apart on a 3x screen! 🤦‍♂️